src/webhooks/registry.ts

```diff
@@ -175,15 +189,23 @@ function buildQuery(
 }
 
 const WebhooksRegistry: RegistryInterface = {
-  webhookRegistry: [],
+  webhookRegistry: {},
+
+  async addHandler({path, topic, webhookHandler}): Promise<void> {
```

Comment:
I think we need to type these params as well:

```suggestion
  async addHandler({path, topic, webhookHandler}: WebhookRegistryEntry): Promise<void> {
```